compose/master/Dockerfile

```diff
@@ -86,6 +86,7 @@ EXPOSE 80
 
 ADD ./mlbench/master/ /app/code/
 
+RUN /venv/bin/python -m pip install djangorestframework
```

Comment:
This line isn't necessary, djangorestframework is in compose/master/requirements.txt